### PR TITLE
plan.sh for libev

### PR DESCRIPTION
**Pull request by bixu**
Signed-off-by: Blake Irvin <blake.irvin@gmail.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
By analyzing the blame information on this pull request, we identified @adamhjk to be a potential reviewer

File: plans/libev/plan.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+pkg_origin=core
+pkg_name=libev
+pkg_version=4.22
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh"
+pkg_license=('MIT')
```
Comment (metadave) [inline plans/libev/plan.sh]: the license appears to be `BSD-2-Clause` and `GPL-2.0`:
http://cvs.schmorp.de/libev/LICENSE?revision=1.11&view=markup

**Comment by metadave** [comment]
looks good, thanks @bixu 

@thesentinels r+

**Comment by thesentinels** [comment]
:pushpin: Commit e1a34d9 has been approved by `metadave`

**Comment by thesentinels** [comment]
:hourglass: Testing commit e1a34d9 with merge 7ac3c98...

**Comment by thesentinels** [comment]
:sunny: Test successful - [travis](https://travis-ci.org/habitat-sh/habitat/builds/139315627)

**Comment by smith** [comment]
Built and uploaded to depot: https://app.habitat.sh/#/pkgs/core/libev/4.22/20160622022208

**Pull request merged**